### ISSUE TITLE
Staging Review finding: Unnecessary `aria-label` may cause issues for voice command/speech recognition users.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91346]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Experience Team 1
**Product:** 21-526EZ
**Feature:** Toxic Exposure Page

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are four mid-flow summary pages for each type of toxic exposure: 
- [toxic-exposure/gulf-war-1990-summary](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/toxic-exposure/gulf-war-1990-summary)
- [toxic-exposure/gulf-war-2001-summary](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/toxic-exposure/gulf-war-2001-summary)
- [toxic-exposure/herbicide-summary](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/toxic-exposure/herbicide-summary)
- [toxic-exposure/additional-exposure-summary](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/toxic-exposure/additional-exposure-summary) 

For each of these, the "Edit locations and dates" link has an `aria-label` that provides extra context for screen reader users, eg. "go back and edit locations and dates for service post-9/11"

I don't think that extra context is necessary for screen reader users. There's only one of these links on the page, and the rest of the page only summarizes your responses to a single grouping of exposures. Likewise, "go back" doesn't add any new information for screen reader users.

Moreover, the `aria-label` introduces an issue for voice command/speech recognition users, eg. Dragon NaturallySpeaking or Apple Voice Control users. 

A sighted voice user may speak a command like "Click Edit locations and dates", with the expected outcome that the link is clicked and navigates the browser. However, voice command software matches user commands against the accessibility tree generated by the browser, and in the accessibility tree the `aria-label` value fully replaces the visible link text.

Behavior isn't super consistent across voice command software. Some software may be smart enough to detect "go back and edit locations and dates..." as a partial match for "Click Edit locations and dates," but others only give you a match if the spoken command matches the beginning of the text string --- meaning, it's listening for "go back and..." But since that text doesn't appear on-screen anywhere, it's unlikely that a user would know to say those words. Voice users typically have other workarounds for clicking a link, but it's some friction that we can avoid fairly easily.

## Recommended action

Since the `aria-label` isn't adding much for screen reader users and introduces an issue for voice users, I think the best option is to just remove it entirely. A regular link with just the visible link text is fine here.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
